.travis.yml

```diff
@@ -26,6 +26,9 @@ before_install:
   if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
     openssl aes-256-cbc -K $encrypted_d8355cc3d845_key -iv $encrypted_d8355cc3d845_iv -in travis.enc -out travis -d
     chmod 0400 travis
+    ## deploy to MeeseeksDev Org's repo
+    openssl aes-256-cbc -K $encrypted_e0a1711769d1_key -iv $encrypted_e0a1711769d1_iv -in deploy_key.enc -out deploy_key -d
```

Comment:
Unfortunately this won't work. You can have only one encrypted file per travis repo, which totally sucks...